index.md

```diff
@@ -1,144 +1,51 @@
+### Current Projects 
+Currently our interns are working on the following six projects in two waves.  These are the original briefs they applied to.  Their work and outputs will be available on our organisation github.
+
+|:---:|--|
+|January 2023|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/CogStack-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TextData-9cf) &emsp; &emsp; &emsp; &emsp; &emsp; |[Enriching Neurology Patient Information using MedCAT](https://nhsx.github.io/nhsx-internship-projects/enriching-neurology-information-medcat/)|
+|![area](https://img.shields.io/badge/Synthetic-brightgreen) &nbsp; ![technique](https://img.shields.io/badge/VAE-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[SynthVAE - Next Steps](https://nhsx.github.io/nhsx-internship-projects/synthvae-extension/)|
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/Hypergraphs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Transforming Healthcare Data with Graph-based Techniques - Next Steps](https://nhsx.github.io/nhsx-internship-projects/transforming-healthcare-data-graph-based-sail-update/)|
+|June 2023|
+|![area](https://img.shields.io/badge/Graphs-yellowgreen) &nbsp; ![technique](https://img.shields.io/badge/GNNs-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet)|[Exploring Data Representations - Graph Neural Networks](https://nhsx.github.io/nhsx-internship-projects/exploring-data-representations-gnn/)|
+|![area](https://img.shields.io/badge/Simulation-darkblue) &nbsp; ![technique](https://img.shields.io/badge/ProcessMining-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/TabularData-green)|[Process Mining to Generate Healthcare Pathways](https://nhsx.github.io/nhsx-internship-projects/process-mining-generate-pathways/)|
+|![area](https://img.shields.io/badge/NLP-yellow) &nbsp; ![technique](https://img.shields.io/badge/Memorisation-lightgrey) &nbsp; ![datamodality](https://img.shields.io/badge/MultiModalData-blueviolet)|[Understanding Privacy Leakage when using Language Models in Healthcare](https://nhsx.github.io/nhsx-internship-projects/language-model-privacy-leakage/)|
+
+<ul class="nhsuk-grid-row nhsuk-card-group">
+  <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+    <div class="nhsuk-card nhsuk-card--clickable">
+        <img class="nhsuk-card__img" src="assets/img/graph.jpg" alt="teaser" />
+        <div class="nhsuk-card__content">
+            <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="projects.html">Projects</a>
+            </h2>
+            <p class="nhsuk-card__description">List of active and open projects to either get involved with or apply to in our next round</p>
```

Comment:
Can we update the image widths/heights(?) to make the text appear at the same point across the two panels?

Reply:
height = "250"